pkg/clusterresourceoverride/mutator.go

```diff
@@ -50,6 +55,40 @@ func (m *podMutator) Mutate(in *corev1.Pod) (out *corev1.Pod, err error) {
 	return
 }
 
+const (
+	SpcType                = "spc_t"
+	SelinuxRelabelResource = "selinuxrelabelfix"
+	SelinuxRelabelGroup    = "admission.node.openshift.io"
+)
+
+var (
+	SelinuxFixEnabledLabelName = fmt.Sprintf("%s.%s/enabled", SelinuxRelabelResource, SelinuxRelabelGroup)
+)
+
+func (m *podMutator) OverrideSelinuxFix(pod *corev1.Pod) {
+	enabled, exists := pod.Labels[SelinuxFixEnabledLabelName]
```

Comment:
Where are we adding the label to the pod? Are namespace labels applied automatically?

Reply:
We aren't adding the label to the pod... I thought the customer is.